### PR TITLE
Fix network kbs

### PR DESCRIPTION
**Pull request by jtowns3**

### CODE REVIEW

**Comment by chrislittle** [comment]
@jtowns3 please review the closed/merged updates from this AM.  https://github.com/CenturyLinkCloud/PublicKB/pulls?q=is%3Apr+is%3Aclosed I saw a ton of errors and did a number of them

**Comment by jtowns3** [comment]
@chrislittle Are the errors fixed now?

File: Network/network-exchange-connectivity-matrix-configuration-guide.md [hunk 1]
```diff
@@ -12,6 +12,16 @@
 
 The table, below, documents the metropolitan areas where Network Exchange is offered and the type and identity of endpoints that are available there for inclusion into an Exchange. Please note any constraints in the following section.
 
+<<<<<<< HEAD
```
Comment (srveit) [inline Network/network-exchange-connectivity-matrix-configuration-guide.md]: Remove merge conflicts. Look for `<<<<<<<`, `=======`, and `>>>>>>>`

File: Network/network-exchange-faqs.md [hunk 2]
```diff
@@ -64,7 +64,11 @@
 
 **Q: Can I view the usage/speed/performance of each of my connections?**
 
+<<<<<<< HEAD
```
Comment (srveit) [inline Network/network-exchange-faqs.md]: Remove merge conflicts. Look for <<<<<<<, =======, and >>>>>>>

**Comment by chrislittle** [comment]
@jtowns3 my pull requests were 
https://github.com/CenturyLinkCloud/PublicKB/pull/1680 (fixed the FAQ line spaces)
https://github.com/CenturyLinkCloud/PublicKB/pull/1682 (fixed the matrix table)

if you have other fixes I didn't touch those but the major ones in this pull request are fixed.  but please review

**Comment by chrislittle** [comment]
@jtowns3 your pull request is all messed up.  you now have double tables as I fixed the table already in your pull request etc.  i'd recommend you close this entire pull request, sync the latest master branch and then view the updates in your editor and fix what is left.

File: Network/network-exchange-connectivity-matrix-configuration-guide.md [hunk 3]
```diff
@@ -12,6 +12,15 @@
 
 The table, below, documents the metropolitan areas where Network Exchange is offered and the type and identity of endpoints that are available there for inclusion into an Exchange. Please note any constraints in the following section.
 
+| Metro Area | Colocation / Dedicated Cloud Compute | Managed Hosting | CenturyLink Cloud |
```
Comment (srveit) [inline Network/network-exchange-connectivity-matrix-configuration-guide.md]: Remove these 9 lines as they are duplicates of the table below.

**Pull request merged**